edb/lib/ext/auth.edgeql

```diff
@@ -72,5 +72,11 @@ CREATE EXTENSION PACKAGE auth VERSION '1.0' {
                 least 32 characters long.";
             set default := "00000000000000000000000000000000";
         };
+
+        create property token_time_to_live -> std::duration {
+            create annotation std::description :=
+                'The time after which an auth token expires.';
+            set default := <std::duration>'336 hours';
```

Comment:
Why was this specific default chosen?  Also, there doesn't seem to be a way to have an unexpiring token (other than setting the ttl to some obscenely large value).

Reply:
No reason, I just thought "2 weeks, sure!" and went with it.

> Also, there doesn't seem to be a way to have an unexpiring token (other than setting the ttl to some obscenely large value).

That's true! Open to suggestion here: is there a way to make this optional, but still have a default if you don't set it?